express-hello-world/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const app = express();
+app.use(function (req, res) {
+  res.send('string');
+});
+
+app.listen(3000, () => {
+});
```

Comment:
It's generally a good idea to print out a message when the server is ready to accept requests. Production servers can take several minutes to start.